src/rnnlm/rnnlm-core-training.cc

```diff
@@ -166,7 +166,7 @@ void RnnlmCoreTrainer::Train(
   ProvideInput(minibatch, derived, word_embedding, &computer);
   computer.Run();  // This is the forward pass.
 
-  ProcessOutput(minibatch, derived, word_embedding,
+  ProcessOutput(true, minibatch, derived, word_embedding,
```

Comment:
it's probably easier to read if you write
`
bool is_backstitch_step = true;
ProcessOutput(is_backstitch_step, ....) 
`

Reply:
Done